packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -0,0 +1,303 @@
+    }
+  ]),
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [
+    withActions,
+    (story: any) =>
+      html`<style>
+          div.slot {
+            width: 24px;
+            height: 24px;
+          }
+          sd-dropdown:not([rounded]) sd-button::part(base),
+          sd-dropdown:not([rounded]) .slot {
+            border-radius: 0;
+          }
+          td.template,
+          .example,
+          div.example {
+            width: 120px;
+            height: 70px;
+          }
+          .example {
+            display: flex;
+            flex-direction: column;
+            padding: 12px;
+          }
+          .template-placement sd-dropdown[placement^='top'] .slot,
+          .template-placement sd-dropdown[placement^='bottom'] .slot {
+            width: 48px;
+          }
+
+          .template-placement sd-dropdown[placement^='left'] .slot,
+          .template-placement sd-dropdown[placement^='right'] .slot {
+            height: 36px;
+          }</style
+        >${story()}`
+  ] as unknown
+};
+
+/**
+ * Default: This shows sd-dropdown in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return html`<div style="height: 200px; overflow: hidden;">
+      ${generateTemplate({
+        constants: {
+          type: 'slot',
+          name: 'default',
+          value: '<div class="example"></div>'
+        },
+        args
+      })}
+    </div>`;
+  }
+};
+
+/**
+ * For round triggers, set the `rounded` attribute. this automatically adds a minimal distance of 1px between the trigger and the panel.
+ */
+
+export const Rounded = {
+  parameters: { controls: { exclude: ['rounded', 'default'] } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        x: {
+          type: 'attribute',
+          name: 'rounded'
+        }
+      },
+      constants: {
+        type: 'slot',
+        name: 'default',
+        value: '<div class="example"></div>'
+      },
+      args
+    })
+};
+
+/**
+ * The overlay automatically adjusts its size to fit the screen. The content gets scrollable then. To disable this behavior, set the `no-auto-size` attribute.
+ */
+
+export const NoAutoSize = {
+  parameters: { controls: { exclude: ['rounded', 'default'] } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        x: {
+          type: 'attribute',
+          name: 'no-auto-size'
+        }
+      },
+      constants: {
+        type: 'slot',
+        name: 'default',
+        value:
+          '<div style="width: 120px;"><div style="height: 110vh; padding: 12px;"><div  class="slot slot--border slot--text slot--background" style="height: 100%; width: auto;">Scroll down<br><br>⬇️</div></div></div>'
+      },
+      args
+    })
+};
+
+/**
+ * The preferred placement of the dropdown can be set with the `placement` attribute.
+ * Note that the actual position may vary to ensure the panel remains in the viewport.
+ */
+
+export const Placement = {
```

Comment:
Unfortunately not. :D This is like the dropdown behaves, that it looks for a way to show up. Otherwise we can't do the screenshot tests. if you have an idea, I would be happy to resolve that? :)

Reply:
strange... the indicator are completely gone now...

Reply:
Nah, we can't do that. So the main purpose is to make a good screenshot test here. Usually the dropdown shouldn't open itself but on a user interaction. Then it tries to be visible as good as possible. Even if it's not visible anymore. Now we open all at once to show the behaviour of the positions. What leads to this... ^^

Reply:
If you have better ideas without breaking the behaviour, let me know! :)

Reply:
Not really. I guess we have to leave it like that then.

Reply:
Did some digging this morning: It works now, at least in the docs! :)

Thanks for requesting changes @karlbaumhauer @MarcMatthiae

Reply:
Fixed for the story as well! :) Thanks for the heads-up